less/index.less

```diff
@@ -17,43 +17,305 @@ article, aside, canvas, details, embed,
 figure, figcaption, footer, header, hgroup, 
 menu, nav, output, ruby, section, summary,
 time, mark, audio, video {
-	margin: 0;
-	padding: 0;
-	border: 0;
-	font-size: 100%;
-	font: inherit;
-	vertical-align: baseline;
+
+// @import url(//"https://fonts.google.com/specimen/IBM+Plex+Mono");
+
+margin: 0;
+padding: 0;
+border: 0;
+font-size: 100%;
+vertical-align: baseline;
 }
+
+
+
 /* HTML5 display-role reset for older browsers */
 article, aside, details, figcaption, figure, 
 footer, header, hgroup, menu, nav, section {
-	display: block;
+display: block;
 }
 body {
-	line-height: 1;
+line-height: 1;
 }
 ol, ul {
-	list-style: none;
+list-style: none;
 }
 blockquote, q {
-	quotes: none;
+quotes: none;
 }
 blockquote:before, blockquote:after,
 q:before, q:after {
-	content: '';
-	content: none;
+content: '';
+content: none;
 }
 table {
-	border-collapse: collapse;
-	border-spacing: 0;
+border-collapse: collapse;
+border-spacing: 0;
 }
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+box-sizing: border-box;
 }
 
 html, body {
-    height: 100%;
+height: 100%;
+width: 80%;
+margin: 0 auto;
+}
+
+// Variables
+
+@header-font: 18px;
+
+// Mixins
+
+.h4Stylz() {
+font-size: 18px;
+
+}
+
+.tdStylz() {
+padding-top: 10px;
+padding-left: 10px;
+}
+
+// Styling for the body
+
+body {
+width: 800px;
+
+@media(max-width: 500px) {
+
+		width: 500px;
+}
+}
+
+
+
+
+// Styling for the Nav element
+
+.navMenu {
+font-size: 12pt;
+
+
+
+nav {
+	display: inline-flex;
+	flex-direction: row;
+	justify-content: space-evenly;
+	margin-left: 55%;
+	width: 50%;
+	padding: 16px;
+}
+
+
+a {
+	text-decoration: none;
+	color: black;
 }
 
+@media (max-width: 500px) {
+	nav {
+		display: flex;
+		flex-direction: column;
+		margin: auto;
+
+		a {
+			font-size: @header-font;
+			height: 70px;
+			width: 500px;
+			text-align: center;
+			padding-right: 350px
+		}
+		
+	}
+}
+}
+
+
+
+// Styling for the header element
```

Comment:
I love that you're doing comments thoughout.  It makes it easier to look through the code!